app/src/org/commcare/dalvik/activities/CommCareSetupActivity.java

```diff
@@ -140,7 +146,7 @@
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         CommCareSetupActivity oldActivity = (CommCareSetupActivity)this.getDestroyedActivityState();
-        this.mFromManager = this.getIntent().
+        this.fromManager = this.getIntent().

```

Comment:
I changed it because no other instance variables in this class use that syntax. My general practice with that has been to just be consistent with whatever is already being done in the class.